includes/admin/class-evf-admin-settings.php

```diff
@@ -308,6 +308,42 @@ class="<?php echo esc_attr( $value['class'] ); ?>"
 						</tr>
 						<?php
 						break;
+					case 'image':
+						$option_value = $value['value'];
+						if ( empty( $option_value ) ) {
+							$option_value = $value['default'];
+						}
+
+						if ( empty( $option_value ) ) {
+							// To be replaced before merge.
+							$option_value = 'https://upload.wikimedia.org/wikipedia/en/thumb/4/41/SELECT_logo.svg/522px-SELECT_logo.svg.png';
```

Comment:
@milan1750 Rather than linking the image we need to save it in our assets folder as a copy.